src/site/markdown/server/manage-webadmin.md

```diff
@@ -1546,53 +1545,10 @@ Response codes:
  - 400: The `fromDomain` resource name is invalid
  - 400: The destination domain specified in the body is invalid
 
-## Creating address regex
```

Comment:
Why do you change the documentation of creating regex mapping in this PR? That should be in https://github.com/linagora/james-project/pull/2526 no?

Reply:
Some copy-paste issues.  Sorry for that.